.github/workflows/e2e.yml

```diff
@@ -28,3 +28,24 @@ jobs:
           account_id: ${{ secrets.COREINT_E2E_ACCOUNT_ID }}
           api_key: ${{ secrets.COREINT_E2E_API_KEY }}
           license_key: ${{ secrets.COREINT_E2E_LICENSE_KEY }}
+  e2eTests-cgroups-v2:
+    # Do not run e2e tests if commit message or PR has skip-e2e.
+    if: ${{ ! contains(github.event.pull_request.labels.*.name, 'ci/skip-e2e') }}
+    runs-on: ubuntu-20.04
```

Comment:
shouldn't it be ubuntu-22.04
?

Reply:
I think so, with ubuntu-20.04 it is running with cgroups v1, isn't it?

Reply:
I do hate copy&paste 😆